app/data_client.go

```diff
@@ -109,6 +109,37 @@ type TabularData struct {
 	TimeReceived  time.Time
 }
 
+// ExportTabularDataReturn represents the result of an ExportTabularData API call.
+type ExportTabularDataReturn struct {
+	OrganizationID   string
+	LocationID       string
+	RobotID          string
+	RobotName        string
+	PartID           string
+	PartName         string
+	ComponentName    string
+	ComponentType    string
+	MethodName       string
+	TimeCaptured     *timestamppb.Timestamp
```

Comment:
We're trying to stay away from proto types, so `TimeCaptured` should be `time.Time`. There are examples on how the conversion happens in other methods.

Reply:
Ah ok, will update. Thanks!